vm/systemSmartContracts/auction.go

```diff
@@ -481,9 +480,13 @@ func (s *stakingAuctionSC) registerBLSKeys(
 
 	for _, blsKey := range newKeys {
 		vmOutput, err := s.executeOnStakingSC([]byte("register@" + hex.EncodeToString(blsKey) + "@" + hex.EncodeToString(registrationData.RewardAddress)))
-		isError := err != nil || vmOutput.ReturnCode != vmcommon.Ok
-		if isError {
-			return nil, err
+		if err != nil {
+			s.eei.AddReturnMessage("cannot do register: error" + err.Error())
```

Comment:
"cannot do register: " + err.Error()  is enough

Reply:
done